30k30days/index.html

```diff
@@ -90,6 +91,36 @@ <h3><strong class="format-money">${{ total }}</strong> Raised of $30,000 Goal ({
         We give students the opportunity to learn directly from successful engineers and entrepreneurs who enjoy sharing knowledge. We're focused on project based learning and aim to teach both core computer science concepts as well as the practical skills and technologies that are required to build web sites and applications today. We currently run classes in San Francisco and Oakland.
       </p>
     </div>
+
+    <div class="row text-center">
+      <h2>Meet our Students</h2>
+      <div class="col-md-3">
+        <h3>Abel Regalado</h3>
+        <div class="video-responsive">
+          <iframe width="560" height="315" src="https://www.youtube.com/embed/33L0e_u1I5s" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
```

Comment:
nit: we should add `?rel=0&modestbranding=1`  to these so we don't get related videos or as much youtube branding

Reply:
good catch.